docs/architecture/decisions/0017-feature-cis-to-retain-shell-scripts.md

```diff
@@ -0,0 +1,31 @@
+# 17. Retain Shell-Based CIS Implementation (No Custom Python Rewrite)
+
+Date: 2025-10-24
+
+## Status
+
+Accepted
+
+## Context
+
+ - The CIS hardening feature in Garden Linux currently executes upstream shell scripts from ovh/debian-cis. 
```

Comment:
it should be explicitly mentioned that this is about the tests in test-ng
it was never up for debate to rewrite the actual implementation of the the feature, only the tests